### PR TITLE
Use consistent title case for headers in Supervisor

### PR DESCRIPTION
**Pull request by maykar**
## Proposed change

This is mostly just to use title case consistently in headers/titles with a few punctuation additions.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by zsarnett — APPROVED** [review]

**Comment by ludeeus** [comment]
Should titles and tooltips end with punctuation?

**Comment by maykar** [comment]
Titles should not and I don't believe I added any punctuation to titles, but you are absolutely right, it isn't needed for tooltips.

File: hassio/src/system/hassio-host-info.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ class HassioHostInfo extends LitElement {
                   ${this.hostInfo.hostname}
                 </span>
                 <mwc-button
-                  title="Change the hostname"
+                  title="Change the hostname."
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: tooltip

File: hassio/src/system/hassio-host-info.ts [hunk 2]
```diff
@@ -87,13 +87,13 @@ class HassioHostInfo extends LitElement {
...
                 <span slot="heading">
-                  IP address
+                  IP Address
                 </span>
                 <span slot="description">
                   ${primaryIpAddress}
                 </span>
                 <mwc-button
-                  title="Change the network"
+                  title="Change the network."
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: tooltip

File: hassio/src/system/hassio-host-info.ts [hunk 3]
```diff
@@ -112,7 +112,7 @@ class HassioHostInfo extends LitElement {
             this.hostInfo.features.includes("hassos")
               ? html`
                   <ha-progress-button
-                    title="Update the host OS"
+                    title="Update the host OS."
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: tooltip

File: hassio/src/system/hassio-host-info.ts [hunk 4]
```diff
@@ -145,7 +145,7 @@ class HassioHostInfo extends LitElement {
           ${this.hostInfo.features.includes("reboot")
             ? html`
                 <ha-progress-button
-                  title="Reboot the host OS"
+                  title="Reboot the host OS."
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: tooltip

File: hassio/src/system/hassio-host-info.ts [hunk 5]
```diff
@@ -156,7 +156,7 @@ class HassioHostInfo extends LitElement {
           ${this.hostInfo.features.includes("shutdown")
             ? html`
                 <ha-progress-button
-                  title="Shutdown the host OS"
+                  title="Shutdown the host OS."
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: tooltip

File: hassio/src/system/hassio-host-info.ts [hunk 6]
```diff
@@ -177,7 +177,7 @@ class HassioHostInfo extends LitElement {
             </mwc-list-item>
             ${this.hostInfo.features.includes("hassos")
               ? html`<mwc-list-item
-                  title="Load HassOS configs or updates from USB"
+                  title="Load HassOS configs or updates from USB."
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: tooltip

File: hassio/src/system/hassio-host-info.ts [hunk 7]
```diff
@@ -221,7 +221,7 @@ class HassioHostInfo extends LitElement {
       });
     } catch (err) {
       showAlertDialog(this, {
-        title: "Failed to get Hardware list",
+        title: "Failed to get hardware list.",
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: Dialog title

File: hassio/src/system/hassio-host-info.ts [hunk 8]
```diff
@@ -249,7 +249,7 @@ class HassioHostInfo extends LitElement {
       // Ignore connection errors, these are all expected
       if (err.status_code && !ignoredStatusCodes.has(err.status_code)) {
         showAlertDialog(this, {
-          title: "Failed to reboot",
+          title: "Failed to reboot.",
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: Dialog title

File: hassio/src/system/hassio-host-info.ts [hunk 9]
```diff
@@ -279,7 +279,7 @@ class HassioHostInfo extends LitElement {
       // Ignore connection errors, these are all expected
       if (err.status_code && !ignoredStatusCodes.has(err.status_code)) {
         showAlertDialog(this, {
-          title: "Failed to shutdown",
+          title: "Failed to shutdown.",
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: Dialog title

File: hassio/src/system/hassio-host-info.ts [hunk 10]
```diff
@@ -307,7 +307,7 @@ class HassioHostInfo extends LitElement {
       await updateOS(this.hass);
     } catch (err) {
       showAlertDialog(this, {
-        title: "Failed to update",
+        title: "Failed to update.",
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: Dialog title

File: hassio/src/system/hassio-host-info.ts [hunk 11]
```diff
@@ -336,7 +336,7 @@ class HassioHostInfo extends LitElement {
         this.hostInfo = await fetchHassioHostInfo(this.hass);
       } catch (err) {
         showAlertDialog(this, {
-          title: "Setting hostname failed",
+          title: "Setting hostname failed.",
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: Dialog title

File: hassio/src/system/hassio-host-info.ts [hunk 12]
```diff
@@ -349,7 +349,7 @@ class HassioHostInfo extends LitElement {
       this.hostInfo = await fetchHassioHostInfo(this.hass);
     } catch (err) {
       showAlertDialog(this, {
-        title: "Failed to import from USB",
+        title: "Failed to import from USB.",
```
Comment (ludeeus) [inline hassio/src/system/hassio-host-info.ts]: Dialog title

File: hassio/src/system/hassio-supervisor-info.ts [hunk 13]
```diff
@@ -184,7 +184,7 @@ class HassioSupervisorInfo extends LitElement {
       this.supervisorInfo = await fetchHassioSupervisorInfo(this.hass);
     } catch (err) {
       showAlertDialog(this, {
-        title: "Failed to set supervisor option",
+        title: "Failed to set supervisor option.",
```
Comment (ludeeus) [inline hassio/src/system/hassio-supervisor-info.ts:187]: Dialog title

File: hassio/src/system/hassio-supervisor-info.ts [hunk 14]
```diff
@@ -200,7 +200,7 @@ class HassioSupervisorInfo extends LitElement {
       this.supervisorInfo = await fetchHassioSupervisorInfo(this.hass);
     } catch (err) {
       showAlertDialog(this, {
-        title: "Failed to reload the supervisor",
+        title: "Failed to reload the supervisor.",
```
Comment (ludeeus) [inline hassio/src/system/hassio-supervisor-info.ts:203]: Dialog title

File: hassio/src/system/hassio-supervisor-info.ts [hunk 15]
```diff
@@ -227,7 +227,7 @@ class HassioSupervisorInfo extends LitElement {
       await updateSupervisor(this.hass);
     } catch (err) {
       showAlertDialog(this, {
-        title: "Failed to update the supervisor",
+        title: "Failed to update the supervisor.",
```
Comment (ludeeus) [inline hassio/src/system/hassio-supervisor-info.ts:230]: Dialog title

File: hassio/src/system/hassio-supervisor-info.ts [hunk 16]
```diff
@@ -257,7 +257,7 @@ class HassioSupervisorInfo extends LitElement {
       await setSupervisorOption(this.hass, data);
     } catch (err) {
       showAlertDialog(this, {
-        title: "Failed to set supervisor option",
+        title: "Failed to set supervisor option.",
```
Comment (ludeeus) [inline hassio/src/system/hassio-supervisor-info.ts:260]: Dialog title

**Review by ludeeus — APPROVED** [review]

**Pull request merged**